hyperon_das_atomdb/database.py

```diff
@@ -29,6 +29,11 @@ class FieldNames(str, Enum):
     IS_TOPLEVEL = 'is_toplevel'
 
 
+class FieldIndexType(str, Enum):
+    DEFAULT = 'default'
```

Comment:
DEFAULT is a bad name as it doesn't give any clue about what the index is. Suggestion: "BINARY_TREE"